pkg/apis/serving/v1alpha1/kfservice_framework_scikit.go

```diff
@@ -17,17 +17,41 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-func (s *ScikitLearnSpec) CreateModelServingContainer(modelName string) *v1.Container {
-	// TODO: https://github.com/kubeflow/kfserving/issues/58
+const (
+	SKLearnEntrypointCommand = "python"
+	SKLearnServingGRPCPort   = "9000"
+	SKLearnServingRestPort   = "8080"
+	SKLearnServingImageName  = "animeshsingh/sklearnserver"
+
+	DefaultSKLearnServingVersion = "latest"
+)
+
+func (s *SKLearnSpec) CreateModelServingContainer(modelName string) *v1.Container {
+	//TODO(@animeshsingh) add configmap for image, default resources, readiness/liveness probe
 	return &v1.Container{
-		Image: "notimplementedyet",
+		Image:     SKLearnServingImageName + ":" + s.RuntimeVersion,
+		Command:   []string{SKLearnEntrypointCommand},
```

Comment:
Can we build the entrypoint correctly in the docker image to avoid requiring a custom command here.

Reply:
Done